package.lisp

```diff
@@ -5,9 +5,6 @@
   (:import-from #:serapeum
                 #:export-always
                 #:->)
-  (:local-nicknames (#:alex #:alexandria)
-                    (#:sera #:serapeum)
-                    (#:file-attributes #:org.shirakumo.file-attributes))
```

Comment:
I think this is related to the uiop version shipped with SBCL (?). I doesn't recognize `local-nicknames`.

Reply:
What's your SBCL and ASDF version? I'd like to keep local nicknames if it isn't much to ask.

hint: https://github.com/lisp-tips/lisp-tips/issues/40

I don't remember when asdf 3.3.5 was released.

Reply:
Current sbcl 2.3.9 and asdf 3.3.1. I rarely remember to update asdf.
I'll update and add back the nicknames.

Reply:
great, thanks.